js/plugins/express/src/index.ts

```diff
@@ -54,6 +54,19 @@ export function expressHandler<
     response: express.Response
   ): Promise<void> => {
     const { stream } = request.query;
+    if (!request.body) {
+      const errMsg =
+        `Error: request.body is undefined. ` +
+        `Are you missing 'content-type: application/json' in your headers? ` +
+        `Or did you forget to use 'express.json()'? `;
+      logger.error(errMsg);
+      response
+        .status(400)
+        .json({ message: errMsg, status: 'BAD REQUEST' })
```

Comment:
is this status correct? `INVALID_ARGUMENT`?

Reply:
done